frontend/src/pages/budgetLines/PreviewTable.jsx

```diff
@@ -11,6 +11,14 @@ import { TotalSummaryCard } from "./TotalSummaryCard";
 import { formatDate } from "../../helpers/utils";
 import "./PreviewTable.scss";
 
+export const loggedInName = (activeUser) => {
```

Comment:
Would it be possible to move this to our helpers file `frontend/src/helpers/utils.js`?

Reply:
Done.